src/main/scala/org/clulab/habitus/apps/HabitusGrounderApp.scala

```diff
@@ -0,0 +1,50 @@
+package org.clulab.habitus.apps
+
+import org.clulab.embeddings.{WordEmbeddingMap, WordEmbeddingMapPool}
+import org.clulab.scala_transformers.tokenizer.Tokenizer
+import org.clulab.scala_transformers.tokenizer.jni.ScalaJniTokenizer
+
+// In this very simple grounder, a Seq of words is being matched with the
+// single word in the embedding map keys that most closely summarizes or
+// approximates them.  Perhaps "male ruler" results in "king".
+class HabitusGrounder(wordEmbeddingMap: WordEmbeddingMap, tokenizer: Tokenizer) {
+
+  def ground(words: Array[String]): String = {
+    val tokens = tokenizer.tokenize(words).tokens
+    val compositeVector = wordEmbeddingMap.makeCompositeVector(tokens)
+    val grounding = wordEmbeddingMap.keys.maxBy { key =>
+      val vector = wordEmbeddingMap.get(key).get
+      val similarity = WordEmbeddingMap.dotProduct(compositeVector, vector)
+
+      println(s"$key\t$similarity")
+      similarity
+    }
+
+    grounding
+  }
+}
+
+object HabitusGrounder {
+  // TODO: A simple "deberta-base" fails!  With v3, the special character is incorrect.
```

Comment:
@RazvanDu, for your embedding_generator/main.py in processors, did you literally use the "microsoft/deberta-base" that is in the [comments](https://github.com/clulab/processors/blob/7f01cd92c43b5aea48963cb6b8510543a99e99f5/main/src/main/python/embedding_generator/main.py#L21) or something else when producing your deberta-embd.tsv?  The deberta-base does not seem to be available at the rust level.

Reply:
That weird, yes, I used the exact values in the comments/example commands and they work just fine.

Reply:
There are some tokenizers that are only directly available to Python, but there is some way to export them.  I just have to remember how that is done.  Thanks!

Reply:
Ah that makes sense, gotcha.